pyoverkiz/models.py

```diff
@@ -82,7 +82,7 @@ class Location:
     latitude: str = field(repr=obfuscate_string, default=None)
     twilight_mode: int
     twilight_angle: str
-    twilight_city: str
+    twilight_city: str = field(repr=obfuscate_string, default=None)
```

Comment:
```suggestion
    twilight_city: str | None = None
```

Reply:
IMO We don’t need to hide this data. In my case, it returns Paris while I live in the south of France.